README.md

```diff
@@ -33,3 +33,15 @@ console.log(obj); // [{ name: 'forgive', value: 'me' }, { name: 'forget', value:
 // delete a cookie
 await cookieStore.delete('forget');
 ```
+
+## Development
+
+### Tests
+
+Before running tests, you'll need to add the following entry to your `/etc/hosts` file on your machine:
+
+```
+127.0.0.1    foo.bar.localhost
```

Comment:
Adding this entry to the /etc/hosts file is necessary to get tests to pass. Without it, tests can't seem to find the `/test/resources/empty_sw.js` file. I'm assuming it can't find it because `foo.bar.localhost` isn't mapped what's normally localhost (127.0.0.1).